mock/py/mockbuild/buildroot.py

```diff
@@ -276,14 +276,23 @@ def all_chroot_packages(self):
         return set(out.splitlines())
 
     @traceLog()
-    def _copy_config(self, filename):
+    def _copy_config(self, filename, symlink=False):
```

Comment:
Is there a good reason this default behavior should be `False`?

Reply:
I am trying to make minimal impact. This function is used to copy /etc/{hosts,resolv.conf} too. In case they are symlinks, I do not want to copy the symlink ('cos the destination will likely does not exists), but I rather want to copy just the content.